lib/pure/parsecfg.nim

```diff
@@ -359,3 +415,139 @@ proc next*(c: var CfgParser): CfgEvent {.rtl, extern: "npc$1".} =
     result.kind = cfgError
     result.msg = errorStr(c, "invalid token: " & c.tok.literal)
     rawGetTok(c, c.tok)
+    rawGetTok(c, c.tok) 
+
+# --------------------- Configuration file related operations ---------------------
+
+proc newCfg*(): OrderedTableRef[string, OrderedTableRef[string, string]] =
```

Comment:
Please define the type `OrderedTableRef[string, OrderedTableRef[string, string]]` above, name it "Config" or something:

```nim
type
  Config = OrderedTableRef[string, OrderedTableRef[string, string]]
```